tests/test_apypie.py

```diff
@@ -38,6 +38,17 @@ def test_http_call_get(api, requests_mock):
     api.http_call('get', '/')
 
 
+def test_http_call_get_headers(api, requests_mock):
+    headers = {'X-Apypie-Test': 'Awesome'}
+    expected_headers = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json;version=1'
+    }
+    expected_headers.update(headers)
```

Comment:
because I only want to one header in `headers` to `http_call`, but it should add the other two other headers (as they are default ones). so I basically expect more than I pass :)

Reply:
oh, i missed the different headers in the 2 lines below :-1: